src/modules/auth/components/login/loginForm/loginForm.tsx

```diff
@@ -28,7 +32,12 @@ export function LoginForm() {
   }
 
   function onSubmit(data: LoginFormData) {
-    console.log(data);
+    axiosInstance
+      .post('/auth/login', data)
```

Comment:
Do not use strings for the endpoints, some of them might change in the process of development. So its better to create a separate constants file for the them. 

```ts
export cons ENDPOINTS = {
  root: '/',
  .../
  } as const
```

Reply:
fixed